edx/edx-enterprise
# [ENT-431]: SuccessFactors Admin Update: Enterprise Customer Configuration

**Description:** 
The Django Admin for SuccessFactors Enterprise Customer Configurations is confusing:

**List View**
1. The Client Secret values should not be displayed on the List view.
2. We can just display checkboxes instead of values for Active and Access Token
3. Fields to display should be Name, Active, Base URL, Access Token, Date Modified, in that order. This will provide a quick visual reference and whether the configuration can generate a valid access token.
4. Please order the configuration records by name ascending, and not by ID descending.

**Detail View:**
1. There is an Enterprise Customer dropdown field, and an "Enterprise customer name" field. Unsure why we have both, so we can probably remove the read-only name field.
2. The Enterprise Customer dropdown field should show the name of the customer, versus the object string
3. The order of the fields should be: Enterprise Customer, Active, Base URL, Company ID, Client ID, Client Secret, User Id, User Type, Has Access Token.
4. SAP Base URL and SAP User Type should be in regular format, not bold
5. All fields should have associated help text

**JIRA:** [ENT-431](https://openedx.atlassian.net/browse/ENT-431)

**Installation instructions:** Simply install edx-enterprise using `saleem-latif/ENT-431` branch.

**Testing instructions:**

1. Open success factors configuration admin page and verify list and details views are according to the description above.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
